app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,68 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions.length">
+  <button
+    uib-dropdown-toggle
+    class="btn btn-link dropdown-toggle"
+    type="button"
+    id="dropdownKebabRight-{{$id}}"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="true">
+    <span class="fa fa-ellipsis-v"></span>
+  </button>
+  <ul
+    class="dropdown-menu dropdown-menu-right"
+    aria-labelledby="dropdownKebabRight">
+    <li
+      ng-repeat="action in notification.actions"
+      role="{{action.isSeparator === true ? 'separator' : 'menuitem'}}"
+      ng-class="{'divider': action.isSeparator === true, 'disabled': action.isDisabled === true}">
+      <a
+        ng-if="!action.isSeparator"
+        class="secondary-action"
+        title="{{action.title}}"
+        ng-click="$ctrl.customScope.handleAction(notification, action)">
+        {{action.name}}
+      </a>
+    </li>
+  </ul>
+</div>
+
+
+<span
+  ng-if="notification.event"
+  class="{{'pull-left ' + $ctrl.customScope.getNotficationStatusIconClass(notification.event)}}"
+  ng-click="$ctrl.customScope.markRead(notification)"></span>
+<div class="drawer-pf-notification-content">
+  <div
+    class="drawer-pf-notification-message"
+    tooltip-append-to-body="true"
+    tooltip-popup-delay="500"
+    tooltip-placement="bottom"
+    tooltip="{{notification.event.message}}"
```

Comment:
Can't decide on the tooltip here... Was this part of the design? Might be better as a `title` attribute.